packages/client/src/v2-events/features/events/components/Review.tsx

```diff
@@ -562,29 +564,56 @@ const ActionContainer = styled.div`
     margin-bottom: 8px;
   }
 `
```

Comment:
More on this when [this ticket](https://github.com/opencrvs/opencrvs-core/issues/8409) will be worked on.